test/jdk/ProblemList.txt

```diff
@@ -722,9 +722,9 @@ java/util/concurrent/ExecutorService/CloseTest.java             8288899 macosx-a
 
 # svc_tools
 
-sun/tools/jstatd/TestJstatdDefaults.java                        8081569,8226420 windows-all
-sun/tools/jstatd/TestJstatdRmiPort.java                         8226420,8251259,8293577 generic-all
-sun/tools/jstatd/TestJstatdServer.java                          8081569,8226420 windows-all
+sun/tools/jstatd/TestJstatdDefaults.java                        8081569 windows-all
+sun/tools/jstatd/TestJstatdRmiPort.java                         8251259,8293577 generic-all
+sun/tools/jstatd/TestJstatdServer.java                          8081569 windows-all
```

Comment:
Why not add [JDK‑8240711], which [JDK‑8226420] is a duplicate of?
```suggestion
sun/tools/jstatd/TestJstatdDefaults.java                        8081569,8240711 windows-all
sun/tools/jstatd/TestJstatdRmiPort.java                         8240711,8251259,8293577 generic-all
sun/tools/jstatd/TestJstatdServer.java                          8081569,8240711 windows-all
```

[JDK‑8226420]: https://bugs.openjdk.org/browse/JDK-8226420 "[JDK‑8226420] sun/tools/jstatd tests failed with Port already in use"
[JDK‑8240711]: https://bugs.openjdk.org/browse/JDK-8240711 "[JDK‑8240711] TestJstatdPort.java failed due to \"ExportException: Port already in use:\""

Reply:
8240711 is fixed, so it cannot be the reason tests are in the problemlist.

8081569 also looks like a duplicate of 8240711 and possibly 8314476, so we should remove that one as well (will do so separately).